.github/workflows/c3-e2e.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       - name: E2E Tests
         uses: ./.github/actions/run-c3-e2e
         env:
-          NPM_PUBLISH_TOKEN: NOT_USED
+          NODE_AUTH_TOKEN: NOT_USED
```

Comment:
Is this needed now?

Reply:
see https://github.com/actions/setup-node/blob/eff380dfbcf941bf8832e4acb788cebe13dfd758/src/authutil.ts#L54

Reply:
probably not, lemme remove it